### DISCUSSION TITLE
Certain buttons on keyboard stopped working

### DISCUSSION THREAD

**Discussion by ThunderVolt-code** [ppy/osu#22922] [Q&A]
### Type

Other

### Bug description

In the menu certain buttons stopped working. Most character buttons were working, but escape and backspace were not. Bug was fixed after game restart

### Version

2023.305.0-lazer

### Logs

[input.log](https://github.com/ppy/osu/files/11009015/input.log)
[network.log](https://github.com/ppy/osu/files/11009016/network.log)
[performance.log](https://github.com/ppy/osu/files/11009017/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11009018/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11009019/updater.log)
[database.log](https://github.com/ppy/osu/files/11009020/database.log)

**Comment by bdach** [comment]
We're going to need more information than this to debug the issue. What was the input device you were using? What were you doing before this happened? Can you reproduce this?
